docs/user_guide.rst

```diff
@@ -2,16 +2,16 @@
 
 .. _user-guide:
 
+##########
 User Guide
-==========
+##########
```

Comment:
Let's do header formatting in a different PR.

How to proceed: checkout this branch (or launch the appropriate codespace), unedit this change, commit the unedited change with `git commit -a --amend --no-edit`, then force push to the branch `git push --force` (you need to make sure you're on the right branch before force pushing! it is dangerous and can potentially overwrite your other PRs. I usually run `git push` and have it show me the error before `git push --force`, chat with me if in doubt).

Reply:
Actually... just do a new commit and don't force push. Unedit the change, commit it, push it. I'll "squash" it when merging so the two commits will just be joined into one.